frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -351,7 +351,7 @@ impl<T: Config> Pallet<T> {
 
```

Comment:
In this pallet, when we create the snapshot, the `desired_winners` part, should we fail, or do something if `desired_winners > T::MaxWinners`? 

This is the condition for the staking miner to not need any tweaks.

Reply:
Taking a minimum of them